bin/zkt

```diff
@@ -2,16 +2,13 @@
 
 cd $(dirname $0)
 
-if which zkup >/dev/null; then
-    cargo uninstall zk_inception
-    cargo uninstall zk_supervisor
+if which zkstackup >/dev/null; then
```

Comment:
This script should be removed; zkstackup should work locally, e.g. as `zkstackup --local`.

Reply:
I still don't know how to find the path to the local installation of zksync_era

Reply:
If you're supplying `--local` flag what prevents you from assuming that it's the directory you're in?

Reply:
yes, we could check, that zkstack folder exists in a folder, we are in. and install it from there

Reply:
yes, we could check, that zkstack folder exists in a folder, we are in. and install it from there

Reply:
Addressed in multiple commits